docs/specs/validation/link-validation.yml

```diff
@@ -482,6 +482,58 @@ outputs:
     {"message_severity":"suggestion","code":"insecure-link","message":"Link 'http://technet.com/index-hero.jpg' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.","file":"file1.md","line":4,"end_line":4,"column":1,"end_column":1}
   file1.json:
 ---
+# Validate dynamic secure link
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rules.json
+    allowlists: allowlist.json
+  rules.json: |
+    {
+      "links": {
+        "rules": [
+          {
+            "type": "SecureLinks",
+            "message": "Link '{0}' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.",
+            "code": "insecure-link-2",
+            "severity": "INFO",
+            "contentTypes": ["conceptual"],
+            "useDynamicDomains": true,
+            "domains": [
```

Comment:
Should we add a case to show that we don't use this `domains` list here?

Reply:
There is one. Look at the 4th line in file1.md, `http://www.google.com`. It is in fixed domains but not in dynamic domains, so it won't be reported.